packages/core/src/components/button/_button.scss

```diff
@@ -16,6 +16,7 @@ Markup:
 
 :disabled - Disabled state
 .pt-disabled - Disabled appearance
+.pt-placeholder - Placeholder appearance
```

Comment:
@llorca Would it make sense to alphabetize these? They seem kind of arbitrary right now.

Reply:
I believe the order is such that it looks good on the example docs

Reply:
Given that, does it make sense where I added it?